tasks/cat2.yml

```diff
@@ -149,21 +149,412 @@
-  template: src=audit.rules.j2 dest=/etc/audit/audit.rules owner=root group=root mode=0640 backup=yes
-  tags: [ 'cat2' , 'V-38580' , 'V-38527' , 'V-38525' , 'V-38522' , 'V-38552' , 'V-38550' , 'V-38557' , 'V-38556' , 'V-38554' , 'V-38559' , 'V-38558' , 'V-38536' , 'V-38540' , 'V-38541' , 'V-38543' , 'V-38547' , 'V-38530' , 'V-38578' , 'V-38575' , 'auditd' ]
-  notify: restart auditd
+      - cat2
+      - patch
+      - medium
+      - V-38461
+      - file_perms
+
+- name: "| MEDIUM | V-38464 | AUDIT |\n
+        The audit system must take appropriate action when there are disk errors on the audit storage volume."
+  shell: grep disk_error_action /etc/audit/auditd.conf | cut -d = -f 2 | xargs basename
+  register:  disk_err_action_audit
+  changed_when: no
+  always_run: yes
+  tags:
+      - cat2
+      - medium
+      - V-38464
+      - auditd
+      - audit
+
+- name: "| MEDIUM | V-38464 | PATCH |\n
+        The audit system must take appropriate action when there are disk errors on the audit storage volume."
+  lineinfile:
+      regexp: '(^disk_error_action\s*=)(\s*)' 
```

Comment:
Syntax error here or the line below potentially. I'm think the backslashes might need to be escaped. i.e "\\s". Also in the next line it needs to be enclosed in double quotes.